.github/workflows/deploy_vz_api.yml

```diff
@@ -0,0 +1,42 @@
+name: "Build & deploy the VZ API"
+
+on:
+  push:
+    branches:
+      - master
+      - production
+    paths:
+      - ".github/workflows/deploy_vz_api.yml"
+      - ".github/workflows/aws-vz-api-helper.sh"
+      - "atd-cr3-api/**"
+  workflow_dispatch:
+    inputs:
+      description:
+        default: ""
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.8"
```

Comment:
this bumps from 3.7 that was used in the CircleCI job. I think this is okay since it is what we used with the same tooling in the Moped workflow.